### PR TITLE
Add feature : Pass through the DNS request if the client does not access the services inside the cluster

### PR DESCRIPTION
**Pull request by arcanique**
…ess the services inside the cluster

**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind feature

**What this PR does / why we need it**:
>
>  Pass through the DNS request if the client does not access the services inside the cluster

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: edgemesh/pkg/server/dns.go [hunk 1]
```diff
@@ -257,19 +272,30 @@ func (q *dnsQuestion) getQName(req []byte, offset uint16) uint16 {
 
 // lookupFromMetaManager implement confirm the service exists
 // for earlier version returns true by default
```
Comment (anyushun) [inline edgemesh/pkg/server/dns.go]: Now we need to delete this line

Comment (arcanique) [reply edgemesh/pkg/server/dns.go]: k

File: edge/pkg/metamanager/msg_processor.go [hunk 2]
```diff
@@ -115,7 +115,7 @@ func parseResource(resource string) (string, string, string) {
 
 // is resource type require remote query
 func requireRemoteQuery(resType string) bool {
-	return resType == model.ResourceTypeConfigmap || resType == model.ResourceTypeSecret || resType == constants.ResourceTypeService || resType == constants.ResourceTypeEndpoints
+	return resType == model.ResourceTypeConfigmap || resType == model.ResourceTypeSecret || resType == constants.ResourceTypeEndpoints
```
Comment (sids-b) [inline edge/pkg/metamanager/msg_processor.go]: We support process remoteQuery for Service. Why are we removing it ?

Comment (arcanique) [reply edge/pkg/metamanager/msg_processor.go]: There are several reasons for the changes here. First，we have to get the service endpoint every time. if we get it from the cloud every time, it is very time consuming and extremely inefficient.  Second, the update of service and endpoint is actually notified by the cloud, we did not take the initiative to watch. Every time we go to the cloud to query, and the related resource information has not changed, the operation of the active query is actually redundant. Therefore, the operation of supporting the service to query on the cloud is removed

Comment (sids-b) [reply edge/pkg/metamanager/msg_processor.go]: OK

**Pull request merged**